Error: Cannot find module 'tslib'

When I import `json-strictify`, I get the following runtime error:

```text
Error: Cannot find module 'tslib'
```

This is caused by `"importHelpers": true` in [`tsconfig.json`](https://github.com/pigulla/json-strictify/blob/develop/tsconfig.json#L8).

Enabling `importHelpers` instructs the TypeScript compiler to emit `require("tslib")` in the generated code. Since `tslib` is not listed as a dependency in `package.json`, this causes the above runtime error to be thrown.

To fix this, either disable `importHelpers`, or install `tslib`.

--- a/package.json
+++ b/package.json
@@ -1,70 +1,71 @@
 {
-    "name": "json-strictify",
-    "version": "6.3.0",
-    "description": "Safely serialize a value to JSON without unintended loss of data or going into an infinite loop due to circular references.",
-    "engines": {
-        "node": ">=10"
-    },
-    "main": "./dist/index.js",
-    "types": "./dist/index.d.ts",
-    "files": [
-        "dist/**/*",
-        "CHANGELOG.md",
-        "README.md",
-        "LICENSE"
-    ],
-    "scripts": {
-        "build": "npm ci && rm -rf dist/ && npm test && tsc --build tsconfig.json",
-        "test": "tsc --noEmit && npm run lint && npm run test-with-coverage && npm audit",
-        "lint": "eslint --ext=.ts 'src/**/*.ts' 'test/**/*.ts'",
-        "mocha": "mocha",
-        "test-with-coverage": "nyc mocha 'test/**/*.spec.ts'",
-        "coveralls": "coveralls < coverage/lcov.info"
-    },
-    "repository": {
-        "type": "git",
-        "url": "https://github.com/pigulla/json-strictify.git"
-    },
-    "keywords": [
-        "JSON",
-        "debugging",
-        "validation",
-        "stringify",
-        "strict"
-    ],
-    "author": "Raphael Pigulla <pigulla@four66.com>",
-    "license": "MIT",
-    "bugs": {
-        "url": "https://github.com/pigulla/json-strictify/issues"
-    },
-    "homepage": "https://github.com/pigulla/json-strictify",
-    "dependencies": {
-        "json-pointer": "^0.6.0",
-        "ts-custom-error": "^3.1.1"
-    },
-    "devDependencies": {
-        "@types/chai": "^4.2.5",
-        "@types/json-pointer": "^1.0.30",
-        "@types/lodash.noop": "^3.0.6",
-        "@types/mocha": "^5.2.7",
-        "@types/node": "^12.12.14",
-        "@typescript-eslint/eslint-plugin": "^2.9.0",
-        "@typescript-eslint/parser": "^2.9.0",
-        "chai": "^4.2.0",
-        "coveralls": "^3.0.9",
-        "eslint": "^6.7.1",
-        "eslint-config-standard": "^14.1.0",
-        "eslint-plugin-import": "^2.18.2",
-        "eslint-plugin-node": "^10.0.0",
-        "eslint-plugin-promise": "^4.2.1",
-        "eslint-plugin-standard": "^4.0.1",
-        "jest": "^24.9.0",
-        "lodash.noop": "^3.0.1",
-        "mocha": "^6.2.2",
-        "nyc": "^14.1.1",
-        "ts-node": "^8.5.2",
-        "tsconfig-paths": "^3.9.0",
-        "type-fest": "^0.8.1",
-        "typescript": "^3.7.2"
-    }
+  "name": "json-strictify",
+  "version": "6.3.0",
+  "description": "Safely serialize a value to JSON without unintended loss of data or going into an infinite loop due to circular references.",
+  "engines": {
+    "node": ">=10"
+  },
+  "main": "./dist/index.js",
+  "types": "./dist/index.d.ts",
+  "files": [
+    "dist/**/*",
+    "CHANGELOG.md",
+    "README.md",
+    "LICENSE"
+  ],
+  "scripts": {
+    "build": "npm ci && rm -rf dist/ && npm test && tsc --build tsconfig.json",
+    "test": "tsc --noEmit && npm run lint && npm run test-with-coverage && npm audit",
+    "lint": "eslint --ext=.ts 'src/**/*.ts' 'test/**/*.ts'",
+    "mocha": "mocha",
+    "test-with-coverage": "nyc mocha 'test/**/*.spec.ts'",
+    "coveralls": "coveralls < coverage/lcov.info"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/pigulla/json-strictify.git"
+  },
+  "keywords": [
+    "JSON",
+    "debugging",
+    "validation",
+    "stringify",
+    "strict"
+  ],
+  "author": "Raphael Pigulla <pigulla@four66.com>",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/pigulla/json-strictify/issues"
+  },
+  "homepage": "https://github.com/pigulla/json-strictify",
+  "dependencies": {
+    "json-pointer": "^0.6.0",
+    "ts-custom-error": "^3.1.1",
+    "tslib": "^1.10.0"
+  },
+  "devDependencies": {
+    "@types/chai": "^4.2.5",
+    "@types/json-pointer": "^1.0.30",
+    "@types/lodash.noop": "^3.0.6",
+    "@types/mocha": "^5.2.7",
+    "@types/node": "^12.12.14",
+    "@typescript-eslint/eslint-plugin": "^2.9.0",
+    "@typescript-eslint/parser": "^2.9.0",
+    "chai": "^4.2.0",
+    "coveralls": "^3.0.9",
+    "eslint": "^6.7.1",
+    "eslint-config-standard": "^14.1.0",
+    "eslint-plugin-import": "^2.18.2",
+    "eslint-plugin-node": "^10.0.0",
+    "eslint-plugin-promise": "^4.2.1",
+    "eslint-plugin-standard": "^4.0.1",
+    "jest": "^24.9.0",
+    "lodash.noop": "^3.0.1",
+    "mocha": "^6.2.2",
+    "nyc": "^14.1.1",
+    "ts-node": "^8.5.2",
+    "tsconfig-paths": "^3.9.0",
+    "type-fest": "^0.8.1",
+    "typescript": "^3.7.2"
+  }
 }

--- a/package.json
+++ b/package.json
@@ -1,6 +1,6 @@
 {
   "name": "json-strictify",
-  "version": "6.3.0",
+  "version": "6.3.1",
   "description": "Safely serialize a value to JSON without unintended loss of data or going into an infinite loop due to circular references.",
   "engines": {
     "node": ">=10"

